CONTRIBUTING.md

```diff
@@ -6,7 +6,7 @@ We are beyond excited to see that you want to contribute! We would love to accep
   - [Contributing to Layer5 Projects](#contributing-to-layer5-projects)
   - [Contributing to Layer5's Blogs](#contributing-to-layer5s-blogs)
   - [How to Contribute](#how-to-contribute)
-      - [Prequisites](#prequisites)
+      - [Prerequisites](#prequisites)
```

Comment:
You will have to fix the `#prequisites` as well.

Reply:
Yeah, would do that!